dracut/30ignition/ignition-reboot.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Ignition (reboot)
+DefaultDependencies=false
+Before=ignition-complete.target
+
+# Make sure ExecStart= runs before we switch root
+Before=initrd-switch-root.target
+
+# Make sure root filesystem is mounted
+After=ignition-mount.service
```

Comment:
Hmm, I think we need `After=ignition-files.service`, so that user-configs can also drop kargs files and have them immediately take effect.

Reply:
Yep, that makes sense!